lib/VisualGenerator.js

```diff
@@ -52,7 +52,7 @@ function generateVisualOptions(visualName) {
         displayName: visualName,
         guid: generateVisualGuid(),
         visualClassName: 'Visual',
-        apiVersion: '1.1.0',
+        apiVersion: '1.2.0',
```

Comment:
Do not change the default API version to 1.2 yet. This is not supported on the portal yet and this would be the "alpha api" 1.1 is the current stable api so it should be the default version when generating a new visual